xbmc/guilib/GUIFontCache.cpp

```diff
@@ -12,14 +12,21 @@
 #include <stdint.h>
 #include <vector>
 
+using namespace std::chrono_literals;
+
+namespace
+{
+constexpr auto FONT_CACHE_TIME_LIMIT = 1000ms;
```

Comment:
`static`

Reply:
@Montellese anonymous namespaces are equivalent to static

Reply:
You're right, I missed the anonymous namespace.